src/app/suggestion-notifications/reciter-suggestions/suggestion-targets/suggestion-targets.state.service.ts

```diff
@@ -0,0 +1,164 @@
+import { Injectable } from '@angular/core';
+
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import {
+  getCurrentUserSuggestionTargetsSelector,
+  getCurrentUserSuggestionTargetsVisitedSelector,
+  getReciterSuggestionTargetCurrentPageSelector,
+  getReciterSuggestionTargetTotalsSelector,
+  isReciterSuggestionTargetLoadedSelector,
+  isReciterSuggestionTargetProcessingSelector,
+  reciterSuggestionTargetObjectSelector
+} from '../selectors';
+import { SuggestionTarget } from '../../../core/suggestion-notifications/reciter-suggestions/models/suggestion-target.model';
+import {
+  ClearSuggestionTargetsAction,
+  MarkUserSuggestionsAsVisitedAction,
+  RefreshUserSuggestionsAction,
+  RetrieveTargetsBySourceAction
+} from './suggestion-targets.actions';
+import { SuggestionNotificationsState } from '../../../notifications/notifications.reducer';
+
+/**
+ * The service handling the Suggestion targets State.
+ */
+@Injectable()
+export class SuggestionTargetsStateService {
+
+  /**
+   * Initialize the service variables.
+   * @param {Store<SuggestionNotificationsState>} store
+   */
+  constructor(private store: Store<SuggestionNotificationsState>) { }
+
+  /**
+   * Returns the list of Reciter Suggestion Targets from the state.
+   *
+   * @return Observable<OpenaireReciterSuggestionTarget>
```

Comment:
This is wrong. There's nothing called "OpenaireReciterSuggestionTarget".  It should be "SuggestionTarget" it appears.